layouts/_default/list.html

```diff
@@ -12,9 +12,21 @@
     {{ end }}
 {{ end }}
 
+{{ define "head-block" }}
```

Comment:
This effect is great, but it has a design issue and an architectural issue:
- Design: this does not work on mobile devices. I think we drop it for now and revisit how we want to launch demos.
- Architecture: list.html would become too complex with this addition. If you're not convinced we should drop this change outright, could we inject it through a partial?